packages/perseus/src/widgets/expression/expression.stories.tsx

```diff
@@ -122,6 +125,16 @@ export const ExpressionItem3 = (args: StoryArgs): React.ReactElement => {
     );
 };
 
+export const AnswerlessExpression = (args: StoryArgs): React.ReactElement => {
```

Comment:
Where is this used? I don't see it referenced in this PR at all.

Reply:
This is how Storybook works, but exporting the function in a `.stories.tsx` file it shows up in SB.

<img width="277" alt="Screenshot 2025-03-21 at 9 23 17 AM" src="https://github.com/user-attachments/assets/17876b1c-597a-4c15-9cb7-2aea224efd0c" />

Reply:
🤦‍♂️  I didn't take note this was in a `.stories.tsx` file!